cli/src/main/java/org/jboss/as/cli/handlers/CommandHandlerWithHelp.java

```diff
@@ -91,8 +91,8 @@ public void handle(CommandContext ctx) throws CommandLineException {
     protected void printHelp(CommandContext ctx) throws CommandLineException {
         InputStream helpInput = WildFlySecurityManager.getClassLoaderPrivileged(CommandHandlerWithHelp.class).getResourceAsStream(filename);
         if(helpInput != null) {
-            BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput));
-            try {
+
+            try (final BufferedReader reader = new BufferedReader(new InputStreamReader(helpInput, StandardCharsets.UTF_8));){
```

Comment:
I've removed all unnecessary try-with-resource changes.
They are only left in few places where the whole open "resource/use it/close it" was used wrongly from beginning. In most cases that is where reader/writer is created outside try block and closed in finally. Only those ware kept with change to try-with-resources, all others ware changed to minimal change needed.